docsite/source/index.html.md

```diff
@@ -10,6 +10,7 @@ sections:
   - arguments
   - options
   - variadic-arguments
+  - commands-with-subcommands-&-params
```

Comment:
Please use `commands-with-subcommands-and-params` instead (same with the corresponding file name).